Add known limitations for Multi Cluster setup (#1382)
keptn/keptn.github.io: content/docs/install/multi-cluster/index.md

| `remoteControlPlane.api.token` | Keptn API token | `""` |

## Known Limitations

> Keptn uses the integration name and version as identifier for connecting the integration to the control plane.

Reviewer: ```suggestion
Each integration plane is assigned an identifier based on the name and version of the Keptn integration service that runs there.
Keptn uses this identifier to connect the Execution Plane to the Control Plane.
```
I'm just guessing at this wording.  The problem I see is that the rest of this section talks about the Execution Plane.  I think you are inferring that the Execution Plane identifier is based on the name of the Keptn service/integration that is installed there.  Maybe this little tidbit should be added to the bullet list at the top of the page that discusses the Execution Plane.  Just need to make the terminology consistent through the page.  It does seem that we may need to add some additional terms to the discussion.

What if one runs multiple Keptn services on an Execution Plane?  Is this identifier formed based on the name of the first service that is installed?

What happens if one runs multiple

Hence, in the case of multiple remote execution planes, each integration must have a different name/version pair, otherwise Keptn will assign the same set of subscriptions.

> To avoid this, we recommend to assign a different name to the integrations.

Reviewer: ```suggestion
To avoid this,  you must assign a different name to each integration.
```
I'm thinking that this is not really optional -- they just have to do it, right?

> Changing the integration name can be done by editing the Distributor `Deployment` manifest and change the environment variable `K8S_DEPLOYMENT_NAME`.

Reviewer: I think we need more explanation about this "Deployment manifest".  K8S_DEPLOYMENT_NAME is listed on the https://keptn.sh/docs/0.18.x/reference/miscellaneous/distributor/ reference page --  is that what you want them to modify?
That page has a section of "Configuration examples" -- maybe we should add a little section about assigning a new name for a multi-cluster environment?
